LoopKit/CarbKit/CarbStore.swift

```diff
@@ -824,6 +824,67 @@ extension CarbStore {
         }
     }
 
+    /// Retrieves a timeline of effect on blood glucose from carbohydrates entered after start date
```

Comment:
There is too much copy paste here. Perhaps the existing getGlucoseEffects could be modified with an additional parameter for foodStart, or a new function added that contains the shared code.  

Also, I'm not sure I understand how only including future carbs would be used in a carb correction notification feature. Could you expound on that a little?

Reply:
The comment for the effectVelocities should be updated to indicate that these are effect velocities of observed insulin counteraction.  Maybe even rename the parameter to better indicate that.

Reply:
Yeah, that copy-and-paste did not look good. I've modified the existing getGlucoseEffects with an additional parameter. 

Below is an example of how this may be used in the Loop carb correction notification algo I am working on. The most common scenario that may require carb correction is when the carbs entered are overestimated or are absorbing slowly for whatever reason. A comparison of insulin counteraction and modeled carb absorption can be used to detect that situation. In a follow-up carb correction check, we may try out the case when past carb entries are "expired". But we need to keep any already entered future carbs. I'll have a lot more details about carb correction algo in a related Loop PR. 
 
![expired-carbs-example](https://user-images.githubusercontent.com/12002177/53847926-c628e780-3f6f-11e9-852e-28efe69ca419.png)